portals/publisher/source/src/app/components/Apis/Details/Policies/PoliciesExpansion.tsx

```diff
@@ -0,0 +1,281 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { FC, useContext, useEffect, useState } from 'react';
+import Grid from '@material-ui/core/Grid';
+import Box from '@material-ui/core/Box';
+import ExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+import Utils from 'AppData/Utils';
+import { FormattedMessage } from 'react-intl';
+import PolicyDropzone from './PolicyDropzone';
+import type { AttachedPolicy, Policy, PolicySpec } from './Types'
+import FlowArrow from './components/FlowArrow';
+import ApiOperationContext from './ApiOperationContext';
+
+interface OPProps {
+    target: any;
+    verb: any;
+    allPolicies: PolicySpec[] | null;
+    isChoreoConnectEnabled: boolean;
+    policyList: Policy[];
+}
+
+const PoliciesExpansion: FC<OPProps> = ({ target, verb, allPolicies, isChoreoConnectEnabled, policyList }) => {
+
+    // Policies attached for each request, response and fault flow
+    const [requestFlowPolicyList, setRequestFlowPolicyList] = useState<AttachedPolicy[]>([]);
+    const [responseFlowPolicyList, setResponseFlowPolicyList] = useState<AttachedPolicy[]>([]);
+    const [faultFlowPolicyList, setFaultFlowPolicyList] = useState<AttachedPolicy[]>([]);
+
+    // Droppable policy identifier list for each request, response and fault flow
+    const [requestFlowDroppablePolicyList, setRequestFlowDroppablePolicyList] = useState<string[]>([]);
+    const [responseFlowDroppablePolicyList, setResponseFlowDroppablePolicyList] = useState<string[]>([]);
+    const [faultFlowDroppablePolicyList, setFaultFlowDroppablePolicyList] = useState<string[]>([]);
+
+
+    const useStyles = makeStyles((theme: any) => {
```

Comment:
Same as above, take the `makeStyles` out from the component, If you want to change style depending on `verb` value, pass it as an argument to `useStyles`

Reply:
fixed with: https://github.com/wso2/apim-apps/pull/177/commits/3be4d1db628b1a5a5e420682698a04bcd29b7f74